ln1_f4.py

```diff
@@ -8,74 +8,171 @@
 import subprocess
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from email.mime.base import MIMEBase
+from email import Encoders
+from openpyxl import Workbook
+from openpyxl.styles import colors, Font, Border, Side
 
+database_file_name = "/home/lax/PycharmProjects/learn1/database4"
+add_database_file_name = "/home/lax/PycharmProjects/learn1/add_database4"
+excel_table_name = "/home/lax/PycharmProjects/learn1/new method test.xlsx"
 
-def get_sysresinfo(command):  # функція виконує консольну команду та повертає результат
+
+def get_sysresinfo(command):  # Executes console command and returns data as list
```

Comment:
Use docstrings in that case - when you describe function functionality.
```
"""Executes console command and returns data as list."""
```
Uses double quotes